bpf/bpf_host.c

```diff
@@ -1272,7 +1272,7 @@ int cil_from_netdev(struct __ctx_buff *ctx)
 	 * ignore the return value from do_decrypt.
 	 */
 	do_decrypt(ctx, proto);
-	if (ctx->mark == MARK_MAGIC_DECRYPT)
+	if (ctx->mark & MARK_MAGIC_DECRYPT)
```

Comment:
I believe the canonical way of doing this is:

```suggestion
	if ((ctx->mark & MARK_MAGIC_HOST_MASK) == MARK_MAGIC_DECRYPT)
```